frontend/src/components/toast/toast-duplicate-input-button.js

```diff
@@ -53,20 +65,32 @@ class ToastDuplicateInputButton extends React.Component {
       display: flex;
       flex-wrap: wrap;
     }
-
+    
     .button {
       display: flex;
       justify-content: center;
       margin-left: -1rem;
     }
+
+    .wrapper {
+      justify-content: flex-end;
+      padding-bottom: 2.3rem;
```

Comment:
try this being 2.25rem or 2.5rem (1rem = 16px so you want the conversion to come out to something that doesn't give you point something pixels) u can use this also https://nekocalc.com/px-to-rem-converter)